pkg/cmd/azurekeyvaulttemplatefuncs.go

```diff
@@ -0,0 +1,97 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azidentity"
+	"github.com/Azure/azure-sdk-for-go/sdk/security/keyvault/azsecrets"
+)
+
+const azureVaultDomain = "vault.azure.net"
+
+type azureKeyVault struct {
+	Name   string `json:"name"  mapstructure:"name"  yaml:"name"`
+	Alias  string `json:"alias" mapstructure:"alias" yaml:"alias"`
+	client *azsecrets.Client
+	cache  map[string]string
+}
+
+func (v *azureKeyVault) URL() string {
+	return "https://" + v.Name + "." + azureVaultDomain + "/"
```

Comment:
This could be simplified to `return fmt.Sprintf("https://%s.vault.azure.net/", v.Name)`.

Reply:
Completed.